From 33a98f2b5343da927f29191348e992f505544873 Mon Sep 17 00:00:00 2001
From: Adam Jackson <ajax@redhat.com>
Date: Wed, 8 Jun 2016 14:28:09 -0400
Subject: [PATCH 9/9] freetype: Fix a logic error in computing face name

gcc6 chirps an indentation warning here, but really this is bad code.
Effectively this would ignore en_US or en_UK names for the font, despite
that those are the English names the font is most likely to have.

Signed-off-by: Adam Jackson <ajax@redhat.com>
Reviewed-by: Alan Coopersmith <alan.coopersmith@oracle.com>
---
 src/FreeType/fttools.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/FreeType/fttools.c b/src/FreeType/fttools.c
index 7c0bd9b..0329fae 100644
--- a/src/FreeType/fttools.c
+++ b/src/FreeType/fttools.c
@@ -100,7 +100,6 @@ FTGetName(FT_Face face, int nid, int pid, int eid, FT_SfntName *name_return)
             case TT_PLATFORM_MICROSOFT:
                 if(name.language_id != TT_MS_LANGID_ENGLISH_UNITED_STATES &&
                    name.language_id != TT_MS_LANGID_ENGLISH_UNITED_KINGDOM)
-                    break;
                     continue;
                 break;
             default:
-- 
2.9.3

